packages/request-logic/specs/request-logic-specification.md

```diff
@@ -362,6 +362,10 @@ This action is valid, if:
 - the Role of the action **signer is the payee**
 - the request state is **NOT** `canceled`
 
+**And if**:
+
+The `balance` is 0 at the time of the cancellation. (see [here](../../advanced-logic/specs/payment-networks-0.1.0.md))
```

Comment:
Request logic doesn't know about the balance.
There is no way to access the balance at this level, so sadly we can't have it at this spec.

Reply:
Ah, you are right. I will put it in payment-networks then, precising that it's a kind of override of the cancellation warnings from the request logic